### PR TITLE
elaborating on the new target service and port selection functionality

### PR DESCRIPTION
**Pull request by robertpatrick**

### CODE REVIEW

File: documentation/staging/content/navigate/kubernetes/k8s-ingress-controller.md [hunk 1]
```diff
@@ -86,7 +86,9 @@ When editing a route:
 - Use the `Name` field to set the route name.  
 - Use the `Virtual Host` and `Path Expression` fields to define the matching rules that determine which requests match this route.  
 - All requests in the defined rules are routed to the service specified by the `Target Service` field that resides in the namespace specified by the read-only
-`Target Service Namespace` field and the port specified by the `Target Port` field. Once the domain has been deployed, you can also select the values from the drop down list. 
+`Target Service Namespace` field and the port specified by the `Target Port` field. Once the domain has been deployed, you can select the `Target Service`
```
Comment (rosemarymarano) [inline documentation/staging/content/navigate/kubernetes/k8s-ingress-controller.md:89]: Once  -> After (from Oracle editors, once – don’t use to mean after; use only to mean less than twice.)

File: documentation/staging/content/navigate/kubernetes/k8s-ingress-controller.md [hunk 2]
```diff
@@ -86,7 +86,9 @@ When editing a route:
...
+ value from the drop-down list of available service in the WebLogic Kubernetes Operator domain's namespace. Once the `Target Service` is selected,
```
Comment (rosemarymarano) [inline documentation/staging/content/navigate/kubernetes/k8s-ingress-controller.md:90]: available service -> available services
Once the `Target Service` is selected -> After the `Target Service` is selected

**Review by rosemarymarano — COMMENTED** [review]
minor edits

File: documentation/staging/content/navigate/kubernetes/k8s-ingress-controller.md (hunk 1 above)
Comment (robertpatrick) [reply documentation/staging/content/navigate/kubernetes/k8s-ingress-controller.md]: fixed

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**